backend/package.json

```diff
@@ -18,23 +19,15 @@
     "socket.io": "^2.1.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.6",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-preset-env": "^1.7.0",
-    "babel-register": "^6.26.0",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "babel-eslint": "^10.0.1",
```

Comment:
Was this also changed to @babel/ or does it remain under the babel- namespace?

Reply:
No, actually looked this up myself - it has not been moved under the babel namespace